### PR TITLE
Recipes ra2

### PR DESCRIPTION
**Pull request by boubou19**
This PR migrates away from the following recipe adders:
- addElectrolyzerRecipe
- addCompressionRecipe
- addCompressorRecipe
- addArcFurnaceRecipe
- addSifterRecipe
- addFormingPressRecipe
- addLaserEngraverRecipe
- addBenderRecipes
- addMixerRecipe
- addThermalCentrifugeRecipe
- addForgeHammerRecipe
- addChemicalBathRecipe
- addFluidCannerRecipe
- addFluidExtractionRecipe
- addChemicalRecipe
- addMultiblockChemicalRecipe
- addCentrifugeRecipe

I reshaped the following files to be a bit more clear:
- ProcessingIngot.java
- ProcessingGem.java
- ProcessingToolOther.java
- ProcessingTransforming.java
- ProcessingWire.java
- ProcessingFood
- ProcessingDye
- ProcessingFoil.java
- GT_Block_Stones_Abstract

Other things i did:
- I added a long override for the duration in GT_RecipeBuilder
- I made the Superconductor -> SuperconductorUHV in material usages
- I removed easier EV+ cables config and usages as it's never enabled in GTNH
- I removed the possibility to enable old chemical recipes as it's not used in GTNH
- I removed Underground Biomes ore classes, we don't use that mod in GTNH

### CODE REVIEW

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingLog.java [hunk 1]
```diff
@@ -97,15 +96,15 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
-            GT_ModHandler.getModItem(Railcraft.ID, "cube", 1L, 8),
-            null,
-            null,
-            null,
-            16,
-            16);
+
+        GT_Values.RA.stdBuilder()
+            .itemInputs(GT_Utility.copyAmount(1L, aStack))
+            .itemOutputs(GT_ModHandler.getModItem(Railcraft.ID, "tile.railcraft.cube", 1L, 8))
```
Comment (chochem) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingLog.java:102]: you broke the item name again that I fixed

Comment (boubou19) [reply src/main/java/gregtech/loaders/oreprocessing/ProcessingLog.java:102]: just cube right?

Comment (chochem) [reply src/main/java/gregtech/loaders/oreprocessing/ProcessingLog.java:102]: yea

**Review by eigenraven — APPROVED** [review]
I'm not sure anyone can review this in full, let's test in game

**Pull request merged**